Gds/src/fprime_gds/common/data_types/ch_data.py

```diff
@@ -128,7 +128,7 @@ def get_str(self, time_zone=None, verbose=False, csv=False):
         if self.val_obj is None:
             ch_val = "EMPTY CH OBJ"
         elif fmt_str:
-            ch_val = fmt_str % (self.val_obj.val)
+            ch_val = fmt_str % self.val_obj.val
```

Comment:
Lets keep the parentheses here for now until GDS is upgraded to use string interpolations into f-string.

Reply:
OK